### PR TITLE
chore: bump driver to 1.11

### PR DESCRIPTION
**Pull request by kblok**

### CODE REVIEW

File: src/Playwright.Tests/ScreencastTests.cs [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
 namespace Microsoft.Playwright.Tests
 {
+    /*
```
Comment (avodovnik) [inline src/Playwright.Tests/ScreencastTests.cs:14]: Should we leave the test, but skip it, and comment the code, so we don't forget?

**Review by avodovnik — COMMENTED** [review]
Also, remove the binary file (FIREFOX/test.png).

File: src/Playwright.Tests/ScreencastTests.cs (hunk 1 above)
Comment (kblok) [reply src/Playwright.Tests/ScreencastTests.cs:14]: I have an issue for that. It will be my next task

**Comment by kblok** [comment]
Images removed and ignored @avodovnik

File: src/Playwright.Tests/ScreencastTests.cs (hunk 1 above)
Comment (avodovnik) [reply src/Playwright.Tests/ScreencastTests.cs:14]: Ok, that's fine then.

File: .gitignore [hunk 2]
```diff
@@ -352,3 +352,6 @@ MigrationBackup/
 src/Playwright.sln.DotSettings
 src/Playwright/Drivers
 src/Playwright/playwright
+src/Playwright.Tests/Screenshots/CHROMIUM/test.png
+src/Playwright.Tests/Screenshots/FIREFOX/test.png
+src/Playwright.Tests/Screenshots/WEBKIT/test.png
```
Comment (avodovnik) [inline .gitignore:357]: I think we could also just say `src/Playwright.Tests/Screenshots/**/*.png` to be a bit more generic here?

Comment (kblok) [reply .gitignore]: Done!
I'm leaving the `test.png` because it's the only one we need to ignore there.

Comment (avodovnik) [reply .gitignore]: Yeah, that's fine. I'm just thinking of how we can minimize changing this in the future :)

**Review by avodovnik — CHANGES_REQUESTED** [review]
Let's look at the failing tests, then we should be ready to go!

**Review by avodovnik — APPROVED** [review]

**Pull request merged**